pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/generator/SegmentGenerationAndPushTaskGenerator.java

```diff
@@ -103,6 +104,21 @@ public String getTaskType() {
     return MinionConstants.SegmentGenerationAndPushTask.TASK_TYPE;
   }
 
+  @Override
+  public int getNumConcurrentTasksPerInstance() {
+    String numConcurrentTasksPerInstance = _clusterInfoAccessor
+        .getClusterConfig(MinionConstants.SegmentGenerationAndPushTask.CONFIG_NUMBER_CONCURRENT_TASKS_PER_INSTANCE);
```

Comment:
maybe use static import ?

Reply:
yeah, I just keep the same convention as in this file. Maybe good to have another PR to swipe all the conventions ;)